### PR TITLE
Update Develop from last weeks release

### PR DESCRIPTION
**Pull request by ElenaFdR**
Update develop after #812.

-----
[View rendered activities/philanthropic-fundraising/index.md](https://github.com/publiccodenet/about/blob/update-from-last-weeks-release/activities/philanthropic-fundraising/index.md)

### CODE REVIEW

**Review by bvhme — APPROVED** [review]

**Comment by bvhme** [comment]
Travis says there is still a link to the file in there in `philanthropic-fundraising`.

File: activities/philanthropic-fundraising/index.md [hunk 1]
```diff
@@ -12,3 +12,4 @@ These resources support our philanthropic fundraising:
 * [our case for support (A4 PDF)](foundation-for-public-code-case-for-support-A4.pdf) explains the impact of making a philanthropic gift to the Foundation for Public Code (also available as a [US Letter PDF](foundation-for-public-code-case-for-support-letter.pdf))
```
Comment (bvhme) [inline activities/philanthropic-fundraising/index.md:12]: Here is the culprit

**Review by bvhme — CHANGES_REQUESTED** [review]

**Comment by bvhme** [comment]
An interesting clash of commits here, I don't really know what happened, but I'm merging this and then we should fix this in `develop`.

**Comment by ElenaFdR** [comment]
Sure, we can merge master into develop - I'll close this PR now. But we still need to fix this on master, which I don't have the rights to do.

**Comment by bvhme** [comment]
This problem does not exist on `master` as far as I can tell, it was just introduced in this branch.

**Pull request closed without merging**